src/Particular.Analyzers/UtcNowAnalyzer.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Particular.Analyzers.Cancellation
+{
+    using System.Collections.Immutable;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Particular.Analyzers.Extensions;
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class UtcNowAnalyzer : DiagnosticAnalyzer
```

Comment:
Nitpick: the other analyzers are named in terms of the thing they analyze, which in this case is calls to `DateTimeOffset.Now` and `DateTime.Now`, so maybe:

```suggestion
    public class DateTimeNowAnalyzer : DiagnosticAnalyzer
```

(I can't think of a collective term for both `DateTime` and `DateTimeOffset`, so I'm using `DateTime` is a suboptimal surrogate.)

Or another idea, which is a bit weird, but maybe works:

```suggestion
    public class NonUtcNowAnalyzer : DiagnosticAnalyzer
```

Reply:
Or `LocalNowUsage`? 🤷